### ISSUE TITLE
[Paola de Perthuis: MyOPE - Malicious security for Oblivious Polynomial Evaluation]

### ISSUE THREAD

**Issue by ninitrava** [protocol/research#40]
### Research seminars are high level talks given by both PL and non-PL employees on topics relevant to the future of computing and internet technology. 
The series launched in September 2019. Talks can be found [here](https://www.youtube.com/watch?v=L4SJzoKHKPk&list=PLhuBigpl7lqu6xWpiXtbEzJQtlMH1tqoG). If you have a suggestion on someone you would like to hear speak, please share below. If you would like to be notified of research talks in the future, please email research@protocol.ai. 

#### Speaker name
- Paola de Perthuis

#### Institution
- ENS Paris
 
#### Suggested topic
- Malicious security for Oblivious Polynomial Evaluation
 
#### Relevant paper (if any)
- MyOPE - Malicious security for Oblivious Polynomial Evaluation - soon on e-print
- Abstract: 
Oblivious Polynomial Evaluation (OPE) schemes are interactive protocols between a sender with a private polynomial and a receiver with a private evaluation point where the receiver learns the evaluation of the polynomial in their point and no additional information. 

We introduce MyOPE, a scheme for OPE in the presence of malicious senders.  MyOPE enforces honest sender behavior and consistency by adding verifiability to the polynomial evaluation.
The main tools used are FHE for input privacy and arguments of knowledge for the verifiability property. 
MyOPE deploys sublinear communication costs in the polynomial degree and a minimal number of rounds of interaction.

In other words, MyOPE can be used as a verifiable computation scheme for polynomial evaluations over FHE ciphertexts. 
MyOPE is best adapted for the unbalanced setting where the degree of the polynomial and the computation power of the sender are large.
MyOPE can therefore be used as a building block in specialized two-party protocols such as Private Set Intersection (PSI), oblivious keyword search, set membership and more. 
As another contribution, we show how to build, from our MyOPE scheme, a PSI protocol that is secure against malicious senders.
 
#### Relevant groups at PL
- Cryptonet Lab, CryptoCompute Lab

**Comment by tjd233** [comment]
scheduled for Sept 28, 21

**Comment by jsoares** [comment]
https://www.youtube.com/watch?v=kpJuKq2bUVM
